awx/ui/src/util/validators.js

```diff
@@ -1,4 +1,4 @@
-import { t } from '@lingui/macro';
+import { plural, t } from '@lingui/macro';
```

Comment:
Hey @AlexSCorey do you want to import lower case plural, instead of Plural component?